client/src/Components/Address/Address.elm

```diff
@@ -1,5 +1,6 @@
 module Components.Address.Address exposing
```

Comment:
I click the suggested address yet still get "unable to find a match for this address":

[Screencast from 08-27-2025 07:02:14 PM.webm](https://github.com/user-attachments/assets/0455fca5-2af4-4fce-84a0-d0d62c345a65)

Reply:
Looks like a bug on their side 🫤, I hope it doesn't happen to other addresses